util/design/lib/OtpMemMap.py

```diff
@@ -412,16 +412,13 @@ class OtpMemMap():
     # This holds the partition/item index dict for fast access.
     part_dict = {}
 
-    def __init__(self, config):
+    def __init__(self, config, seed):
```

Comment:
Is it correct that if the `seed` argument wasn't provided, the script would throw an error?

Reply:
Correct!